test/java/seedu/addressbook/commands/SortCommandTest.java

```diff
@@ -0,0 +1,47 @@
+package seedu.addressbook.commands;
+
+import org.junit.Before;
+import org.junit.Test;
+import seedu.addressbook.data.AddressBook;
+import seedu.addressbook.data.person.*;
+import seedu.addressbook.data.tag.UniqueTagList;
+import seedu.addressbook.util.TestUtil;
+
+import java.util.*;
+
+import static org.junit.Assert.assertTrue;
+
+public class SortCommandTest {
+
+    private AddressBook addressBook;
+
+    @Before
+    public void setUp() throws Exception {
+        Person C = new Person(new Name("CCC"), new Phone("61234567", false),
+                new Email("c@doe.com", false), new Address("395C Ben Road", false), new UniqueTagList());
+        Person A = new Person(new Name("AAA"), new Phone("91234567", false),
+                new Email("a@doe.com", false), new Address("33G Ohm Road", false), new UniqueTagList());
+        Person D = new Person(new Name("DDD"), new Phone("63345566", false),
+                new Email("d@doe.com", false), new Address("55G Abc Road", false), new UniqueTagList());
+        Person B = new Person(new Name("BBB"), new Phone("61121122", false),
+                new Email("b@grant.com", false), new Address("44H Define Road", false),
+                new UniqueTagList());
+
+        addressBook = TestUtil.createAddressBook(C, A, D, B);
+    }
+
+    /**
+     * Executes the command, and checks that the execution does not throw any Exception.
+     */
+    @Test
+    public void testSortCommand() throws Exception{
```

Comment:
Won't really go through your JUnit yet since you will only really learn it next week, but try to follow the current method naming syntax like `execute_command()`, etc.

Reply:
@Wuhao-ooo I am wrong -- do not follow the syntax i provided, it has now been outdated. Follow point 5 of our coding standards: https://github.com/oss-generic/process/blob/master/codingStandards/CodingStandard-Java.adoc#naming-conventions

Reply:
Ok! Thanks.